eventlet/zipkin/http.py

```diff
@@ -12,45 +11,14 @@
 HDR_SAMPLED = 'X-B3-Sampled'
 
 
-if six.PY2:
-    __org_endheaders__ = httplib.HTTPConnection.endheaders
-    __org_begin__ = httplib.HTTPResponse.begin
-
-    def _patched_endheaders(self):
-        if api.is_tracing():
-            trace_data = api.get_trace_data()
-            new_span_id = api.generate_span_id()
-            self.putheader(HDR_TRACE_ID, hex_str(trace_data.trace_id))
-            self.putheader(HDR_SPAN_ID, hex_str(new_span_id))
-            self.putheader(HDR_PARENT_SPAN_ID, hex_str(trace_data.span_id))
-            self.putheader(HDR_SAMPLED, int(trace_data.sampled))
-            api.put_annotation('Client Send')
-
-        __org_endheaders__(self)
-
-    def _patched_begin(self):
-        __org_begin__(self)
-
-        if api.is_tracing():
-            api.put_annotation('Client Recv (%s)' % self.status)
-
-
 def patch():
-    if six.PY2:
```

Comment:
I wonder if it would be more convenient to remove this entire module. I'm not sure, but the warning message below seems to suggest that zipkin is not supported by Python 3+.

Thoughts?

Reply:
I wondered about that but lacked confidence to delete it outright since it might for example have a TODo intention to port it.

Reply:
I've no idea if someone else wanted to port it.
I'd personally suggest to remove it, but, maybe we want some deprecation period first, though, if I'm right, it won't work with python 3...